programs/bpf_loader/Cargo.toml

```diff
@@ -21,6 +21,9 @@ solana-runtime = { path = "../../runtime", version = "=1.7.0" }
 solana-sdk = { path = "../../sdk", version = "=1.7.0" }
 solana_rbpf = "=0.2.8"
 thiserror = "1.0"
+hkdf = "0.11.0"
```

Comment:
nit: nice to keep these in alphabetical order

Reply:
Fixed in 03af7519e1ca7a29ed2ee25c2cd6b31e0d4e99fd